portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,14 +38,25 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Could you add a comment on what this function does exactly?

Reply:
Done, can you tell me what do you think about the comment? I'm not sure if it is a good one...